torch/csrc/autograd/python_hook.cpp

```diff
@@ -51,7 +51,7 @@ auto PyFunctionPreHook::operator()(const variable_list& values) -> variable_list
   }
 
   variable_list results(values);
-  results[value_idx] = ((THPVariable*)value.get())->cdata;
+  if (THPVariable_Check(value)) results[value_idx] = ((THPVariable*)value.get())->cdata;
```

Comment:
It would be better to use a cheaper check for `Py_None` here I think

Reply:
I see, I will change it back